From 3d05fc41840219cf7d00172a5a09cbf6ba607d98 Mon Sep 17 00:00:00 2001
From: jbelloua <jean-marie.bellouard@intel.com>
Date: Fri, 5 Oct 2012 17:30:41 +0200
Subject: [PATCH 05/49] [PORT FROM MAIN] [Audio] Add FM RX stream type in
 audio FWK

BZ: 46452

Add new stream in enum

Change-Id: Ic65d2fd189546cfc46e7f4ca6b1af66617549f10
Orig-Change-Id: I8cbc9d3eed14756b2df6877cd3b5837455698a47
Signed-off-by: jbelloua <jean-marie.bellouard@intel.com>
Reviewed-on: http://android.intel.com:8080/75601
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/system/audio.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index d09910b..40530f6 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -46,6 +46,7 @@ typedef enum {
     AUDIO_STREAM_ENFORCED_AUDIBLE = 7, /* Sounds that cannot be muted by user and must be routed to speaker */
     AUDIO_STREAM_DTMF             = 8,
     AUDIO_STREAM_TTS              = 9,
+    AUDIO_STREAM_FM_RX            = 10,
 
     AUDIO_STREAM_CNT,
     AUDIO_STREAM_MAX              = AUDIO_STREAM_CNT - 1,
-- 
1.7.9

